core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -215,6 +215,25 @@
   protected static final String ENABLE_GZIP_COMPRESSION_DOC = "Enable gzip compression";
   private static final boolean ENABLE_GZIP_COMPRESSION_DEFAULT = true;
 
+  public static final String REST_SERVLET_INITIALIZERS_CLASSES_CONFIG =
+      "rest.servlet.initializor.classes";
+  public static final String REST_SERVLET_INITIALIZERS_CLASSES_DOC =
+      "Defines one or more initializer for the rest endpoint's ServletContextHandler. "
+          + "Each initializer must implement Consumer<ServletContextHandler>. "
```

Comment:
I wonder if we should create an explicit interface that extends `Consumer<ServletContextHandler>`. The benefit would be twofold:
- any type mismatch will be handled with an explicit error message from `RestConfig.getConfiguredInstances(...)` (we currently don't have a way of enforcing the generic type)
- If the interface ever needs to evolve, any plugin that needs to be updated will be caught at compile time rather than runtime.

Reply:
Hey @xvrl, as I mentioned in this PR's blurb at the top, I've deliberately _not_ used a custom interface, but a standard Java one, so that there are no rest-util dependencies needed.  

However, if you want, I can create a new interface and stick it in a new module in this repo.